README.md

```diff
@@ -68,6 +68,15 @@ const App = () => {
           orientation: 'portrait',
         }}
         liveStreamKey="your-livestrem-key"
+        onConnectionSuccess={() => {
+          //do what you want
+        }}
+        onConnectionFailed={(e) => {
+          //do what you want
+        }}
+        onDisconnect={() => {
+          //do what you want
```

Comment:
Could you add a log in the connection callbacks (and display parameters if applicable)?